src/main/java/nl/tudelft/ti2206/group9/level/State.java

```diff
@@ -20,6 +22,11 @@
 	/** Highest score ever obtained. */
 	private static double highscore;
 
+	/** Skin to be used. */
+	private static AbstractSkin skin;
+	/** Flag for default skin. */
+	private static boolean defaultSkin = true;
```

Comment:
Instead of defining a defaultSkin flag, you could set `skin = Style.getNoob()` in the `State.resetAll()` method.

<review, logic